src/main/java/frc/robot/commands/powerCellScoringCommands/TurnToTopTargetCommand.java

```diff
@@ -20,9 +20,8 @@
     private final double MIN_SPEED = 0.05;
     final double KpAIM = 0.025;
 
-    public TurnToTopTargetCommand(Drive drive, Limelight limelight) {
```

Comment:
You cannot get rid of the limelight subsystem in this class.  It is needed to turn to target.

Reply:
I added it back earlier today